### PR TITLE
Laravel Docs Extension

### PR DESCRIPTION
**Pull request by indykoning**
## Description

This extension will allow you to search the [Laravel Docs](https://laravel.com/docs/8.x) from Raycast

I built this extension using the [Algolia Extension](https://github.com/raycast/extensions/tree/main/extensions/algolia) as a base in order to keep code style the same.

## Type of change

- New extension

## Screencast

image

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Hi @indykoning 👋 

Just a quick question – are we allowed to use Laravel's Algolia search 😄

**Comment by pernielsentikaer** [comment]
Is there a reason why #605 / #606 / #607 are trying to achieve the same and are created within one hour of each other? I'm confused here.

**Comment by indykoning** [comment]
> Is there a reason why #605 / #606 / #607 are trying to achieve the same and are created within one hour of each other? I'm confused here.

Haha I think this might already be one of the biggest coincidences of the year.
I had started working on this a couple hours earlier because I didn't see anyone making one yet.

And I'm guessing they've done the same.

They all seem to try and achieve it in the same manner.

File: extensions/laravel-docs/package.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "laravel-docs",
+  "title": "Laravel Docs",
+  "description": "This extension allows you to search through the Laravel docs directly from Raycast.",
+  "icon": "command-icon.png",
+  "author": "indykoning",
```
Comment (pernielsentikaer) [inline extensions/laravel-docs/package.json:7]:
```suggestion
  "author": "indykoning",
  "contributors": [
    "aglipanci",
    "eth0"
  ],
```

**Comment by pernielsentikaer** [comment]
Do you have any feedback @Erth0 / @aglipanci that we might look into before merging or is everything awesome 💪

**Comment by aglipanci** [comment]
Looks good to me! Great job @indykoning 🚀

**Review by pernielsentikaer — APPROVED** [review]
Awesome @indykoning – let's get it out there!

I'm going to accept it and then will my colleague do a final review before it's finally merged.

**Pull request merged**